ID=4     : size=70474     : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.osapi compute.wsgi.server <:*:> - -] <:IP:> <:*:> <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=10    : size=14700     : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO <:*:> <:*:> - - - - -] <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=2     : size=12796     : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> <:*:> nova.virt.libvirt.imagecache [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] <:*:> base <:*:> /var/lib/nova/instances/ base/<:SEQ:>
ID=6     : size=11214     : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.claims <:*:> - -] [instance: <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=50    : size=7503      : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.osapi compute.wsgi.server <:*:> - -] <:IP:> <:*:> <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=7     : size=7476      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=11    : size=7226      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.imagecache [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] image <:DURATION:>d71-34c5-4fbb-86c4-40623fbe45b4 at (/var/lib/nova/instances/ base/<:SEQ:>): in use: on this node <:NUM:> local, <:NUM:> on other nodes sharing this instance storage
ID=17    : size=6425      : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server <:*:> - - - - -] <:IP:>,<:IP:> "GET <:*:> <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=46    : size=5607      : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO <:*:> <:*:> - - - - -] <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=48    : size=5357      : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> <:*:> nova.virt.libvirt.imagecache [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] <:*:> base <:*:> /var/lib/nova/instances/ base/<:SEQ:>
ID=47    : size=4817      : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.imagecache [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] image <:DURATION:>d71-34c5-4fbb-86c4-40623fbe45b4 at (/var/lib/nova/instances/ base/<:SEQ:>): in use: on this node <:NUM:> local, <:NUM:> on other nodes sharing this instance storage
ID=20    : size=4454      : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server [-] <:IP:>,<:IP:> "GET <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=19    : size=3878      : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server [-] <:IP:>,<:IP:> "GET <:*:> <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=14    : size=3736      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.driver [-] [instance: <:*:> Instance <:*:> successfully.
ID=15    : size=3736      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager <:*:> - -] [instance: <:*:> Took <:NUM:>.<:NUM:> seconds to <:*:> the instance on the hypervisor.
ID=21    : size=3736      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.driver <:*:> - -] [instance: <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=18    : size=3732      : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server <:*:> - - - - -] <:IP:>,<:IP:> "GET <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=12    : size=3600      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager [req-3ea4052c-<:DURATION:>-4b64-9e<:DURATION:>-<:DURATION:>64c<:DURATION:>94ab - - - - -] [instance: <:*:> During sync power state the instance has a pending task (spawning). Skip.
ID=3     : size=3140      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO <:*:> <:*:> <:*:> - - - <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=8     : size=2544      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.resource tracker [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=1     : size=1869      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager <:*:> - -] [instance: <:*:> Took <:NUM:>.<:NUM:> seconds to deallocate network for instance.
ID=5     : size=1869      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.claims <:*:> - -] [instance: <:*:> Attempting claim: memory <:NUM:> MB, disk <:NUM:> GB, vcpus <:NUM:> CPU
ID=13    : size=1868      : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.api.openstack.compute.server external events <:*:> - -] Creating event <:*:> for instance <:*:>
ID=16    : size=1868      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager <:*:> - -] [instance: <:*:> Took <:NUM:>.<:NUM:> seconds to build instance.
ID=22    : size=1868      : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.api.openstack.wsgi <:*:> - -] HTTP exception thrown: No instances found for any event
ID=9     : size=1272      : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.resource tracker [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Final resource view: name=cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us phys ram=64172MB used <:*:> phys disk=15GB used <:*:> total vcpus=<:NUM:> used vcpus=<:NUM:> pci stats=[]
ID=72    : size=1176      : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.claims <:*:> - -] [instance: <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=55    : size=934       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.resource tracker [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=24    : size=921       : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.imagecache [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Base or swap file too young to remove: /var/lib/nova/instances/ base/<:SEQ:>
ID=65    : size=786       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=58    : size=668       : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server <:*:> - - - - -] <:IP:>,<:IP:> "GET <:*:> <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=57    : size=663       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO <:*:> <:*:> <:*:> - - - <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=59    : size=473       : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server [-] <:IP:>,<:IP:> "GET <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=56    : size=467       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.resource tracker [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Final resource view: name=cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us phys ram=64172MB used <:*:> phys disk=15GB used <:*:> total vcpus=<:NUM:> used vcpus=<:NUM:> pci stats=[]
ID=51    : size=394       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.driver [-] [instance: <:*:> Instance <:*:> successfully.
ID=52    : size=394       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager <:*:> - -] [instance: <:*:> Took <:NUM:>.<:NUM:> seconds to <:*:> the instance on the hypervisor.
ID=68    : size=392       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.driver <:*:> - -] [instance: <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=61    : size=391       : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server [-] <:IP:>,<:IP:> "GET <:*:> <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=53    : size=384       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager [req-3ea4052c-<:DURATION:>-4b64-9e<:DURATION:>-<:DURATION:>64c<:DURATION:>94ab - - - - -] [instance: <:*:> During sync power state the instance has a pending task (spawning). Skip.
ID=62    : size=382       : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.metadata.wsgi.server <:*:> - - - - -] <:IP:>,<:IP:> "GET <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=26    : size=335       : nova-scheduler.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.scheduler.host manager <:*:> - - - - -] The instance sync for host 'cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us' did not match. Re-created its InstanceList.
ID=23    : size=299       : nova-scheduler.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.scheduler.host manager <:*:> - - - - -] Successfully synced instances from host 'cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us'.
ID=25    : size=258       : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.osapi compute.wsgi.server <:*:> - -] <:IP:> "GET /v2/<:SEQ:>/servers/detail?all <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=63    : size=203       : nova-scheduler.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.scheduler.host manager <:*:> - - - - -] Successfully synced instances from host 'cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us'.
ID=54    : size=198       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager <:*:> - -] [instance: <:*:> Took <:NUM:>.<:NUM:> seconds to build instance.
ID=49    : size=197       : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.api.openstack.compute.server external events <:*:> - -] Creating event <:*:> for instance <:*:>
ID=69    : size=196       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager <:*:> - -] [instance: <:*:> Took <:NUM:>.<:NUM:> seconds to deallocate network for instance.
ID=70    : size=196       : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.api.openstack.wsgi <:*:> - -] HTTP exception thrown: No instances found for any event
ID=71    : size=196       : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.claims <:*:> - -] [instance: <:*:> Attempting claim: memory <:NUM:> MB, disk <:NUM:> GB, vcpus <:NUM:> CPU
ID=60    : size=95        : nova-api.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.osapi compute.wsgi.server <:*:> - -] <:IP:> "GET /v2/<:SEQ:>/servers/detail?all <:*:> HTTP/<:NUM:>.<:NUM:>" status: <:NUM:> len: <:NUM:> time: <:NUM:>.<:NUM:>
ID=74    : size=74        : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.virt.libvirt.imagecache [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Base or swap file too young to remove: /var/lib/nova/instances/ base/<:SEQ:>
ID=29    : size=65        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager [-] [instance: <:*:> During sync power state the instance has a pending task <:*:> Skip.
ID=28    : size=41        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> WARNING nova.compute.manager [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] While synchronizing instance power states, found <:NUM:> instances in the database and <:NUM:> instances on the hypervisor.
ID=34    : size=40        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=32    : size=32        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:> <:*:>
ID=33    : size=32        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task <:*:> <:*:>
ID=73    : size=29        : nova-scheduler.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.scheduler.host manager <:*:> - - - - -] The instance sync for host 'cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us' did not match. Re-created its InstanceList.
ID=31    : size=24        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task <:*:> <:*:> <:*:> <:*:> <:*:>
ID=27    : size=22        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Running instance usage audit for host cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us from <:DATE:> <:TIME:> to <:DATE:> <:TIME:>. <:NUM:> instances.
ID=42    : size=11        : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> WARNING nova.virt.libvirt.driver [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] couldn't obtain the vcpu count from domain id: <:*:> exception: Requested operation is not valid: cpu affinity is not supported
ID=30    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Error during ComputeManager. run image cache manager pass
ID=35    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task File "/usr/lib/python2.<:NUM:>/dist-packages/nova/virt/libvirt/imagecache.py", line <:NUM:>, in update
ID=36    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task File "/usr/lib/python2.<:NUM:>/dist-packages/nova/virt/libvirt/utils.py", line <:NUM:>, in get disk backing file
ID=37    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task InvalidDiskInfo: Disk info file is invalid: qemu-img failed to execute on <:*:> : Unexpected error while running command.
ID=38    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task Command: /usr/bin/python -m oslo concurrency.prlimit --as=<:NUM:> --cpu=<:NUM:> -- env LC ALL=C LANG=C qemu-img info <:*:>
ID=39    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task Exit code: <:NUM:>
ID=40    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task Stderr: u"qemu-img: Could not open <:*:> Could not open <:*:> No such file or directory\n"
ID=41    : size=8         : nova-compute.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR oslo service.periodic task
ID=64    : size=8         : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] Running instance usage audit for host cp-<:NUM:>.slowvm1.tcloud-pg0.utah.cloudlab.us from <:DATE:> <:TIME:> to <:DATE:> <:TIME:>. <:NUM:> instances.
ID=66    : size=4         : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> WARNING nova.compute.manager [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] While synchronizing instance power states, found <:NUM:> instances in the database and <:NUM:> instances on the hypervisor.
ID=67    : size=4         : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> INFO nova.compute.manager [-] [instance: <:*:> During sync power state the instance has a pending task (spawning). Skip.
ID=43    : size=2         : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> ERROR keystonemiddleware.auth token <:*:> - -] Bad response code while validating token: <:NUM:>
ID=44    : size=2         : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> WARNING keystonemiddleware.auth token <:*:> - -] Identity response: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML <:NUM:>.<:NUM:>//EN">
ID=45    : size=2         : nova-api.log.<:NUM:>.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> CRITICAL keystonemiddleware.auth token <:*:> - -] Unable to validate token: Failed to fetch token data from identity server
ID=75    : size=1         : nova-compute.log.<:DATE:> <:TIME:> <:DATE:> <:TIME:>.<:NUM:> <:NUM:> WARNING nova.virt.libvirt.driver [req-addc1839-2ed5-<:NUM:>-b57e-5854eb7b8b09 - - - - -] couldn't obtain the vcpu count from domain id: b<:DURATION:><:NUM:>-c79b-41fe-<:NUM:>-5f2b89b<:DURATION:><:NUM:>, exception: Requested operation is not valid: cpu affinity is not supported
